app/Http/Controllers/RegisterController.php

```diff
@@ -9,7 +9,7 @@
 class RegisterController extends Controller
 {
     public function showForm(){
-        return view('authentication.register');
+        return view('authentication\register');
```

Comment:
I think it should be: authentication/register

Reply:
Ok